spacialgaze-plugins/misc.js

```diff
@@ -542,4 +542,11 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox("<a href=\"https://discord.gg/cwfAqdN\">The Official SpacialGaze Discord</a>");
 	},
+	'!digidex': true,
+	dd: 'digidex',
+	digidex: function (target, room, user) {
+		if (!target) return this.parse("/help digidex");
```

Comment:
what about the ability to broadcast? if thats possible at all this way

Reply:
if (cmd.broadcast)?

Reply:
i think `if (!this.runBroadcast()) reutrn;` will work as normal, idk because of parse though

Reply:
Uhhh for abilities it could still be treated as a /dt

Reply:
try making digidex broadcastable is what im asking :P

Reply:
Ohhh

Reply:
It wouldn't ever broadcast because it's explicitly running `/dt`, and not checking the command token. A possible workaround could be `this.parse(\`${this.cmdToken}dt ${target}, digimon`\)`

Reply:
I still am questioning how to go about this

Reply:
Figured it out